.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,161 @@
+#
+# Copyright (C) 2015 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Bug report 🐞
+description: Report a bug found in Fabric8 kubernetes-client
+labels: [kind/bug]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Creating a new Bug 🐞
+        🔍  Before opening a new issue please search existing issues at https://github.com/fabric8io/kubernetes-client/issues
+
+        🤔  To make it easier for us to help you, please include as much useful information as possible.
+
+        Useful Links:
+
+          - 📄  Documentation: https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md
+          - 📝  Contributing: https://github.com/fabric8io/kubernetes-client/blob/master/CONTRIBUTING.md
+
+
+  - type: textarea
+    id: describe-bug
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: Describe the bug
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Fabric8 Kubernetes Client version
+      description: Fabric8 Kubernetes Client version (can be found in pom.xml dependency or build.gradle implementation section)
+      options:
+        - "5.10.1@latest"
+        - "next (development version)"
+        - "5.10.0"
+        - "5.9.0"
+        - "5.8.0"
+        - "5.7.0"
+        - "5.6.0"
+        - "5.5.0"
+        - "5.4.0"
+        - "5.3.0"
+        - "5.2.0"
+        - "5.1.0"
+        - "other (please specify in additional context)"
```

Comment:
This dropdown will become outdated very soon.
We can keep it, but we'll probably need to figure out a way to automate something to make it self-updatable

Reply:
@manusa - I am not sure about dynamically passing values to it. We can remove this version section completely if not needed. We referred it from [Eclipse Che.](https://github.com/eclipse/che/blob/main/.github/ISSUE_TEMPLATE/bug_report.yml)

Reply:
maybe our release scripts could update that yaml file adding the version released.

Reply:
yes, absolutely, that is something we have to look for.